ui/imports/shared/controls/TransactionAddressTile.qml

```diff
@@ -0,0 +1,91 @@
+import QtQuick 2.13
+import QtQuick.Layouts 1.15
+import QtQuick.Shapes 1.15
+
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+
+/*!
+   \qmltype TransactionAddressTile
+   \inherits TransactionActivityTile
+   \inqmlmodule shared.controls
+   \since shared.controls 1.0
+   \brief It displays list of addresses for wallet activity.
+
+   The \c TransactionAddressTile can display list of addresses formatted in specific way.
+
+   \qml
```

Comment:
the naming is a bit difficult to process as all of these only demonstrate Tx addres using it sses right?
Also I don't see the usage of this in the "current" Transaction Details View, so if you could do that, would be great.
Also having these in storybbook will help cause thats how we are heading ahead with implementation right now

Reply:
I went with naming used there already - InformationTile.
My decision to create base component because we will be able to reuse it in whole screen.

Please look as an example here: https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?type=design&node-id=12689-341125&t=V0cyIC3m2qoRmcTU-0

Below addresses the "tiles" are have the same look, just different contents.

Only thing I think could make sense is to merge "TransactionAddress" with "TransactionAddressTile", but reason for having them independent is to be able to test them.